circuit/js/src/constants.ts

```diff
@@ -15,9 +15,9 @@ export const COMPUTE_NUM_INSTANCES = USER_COMPUTE_NUM_INSTANCES + SUBQUERY_NUM_I
 
 export const DEFAULT_CIRCUIT_CONFIG = {
```

Comment:
had to update these constants as well. i couldn't find any tools constants for the other values; let me know if we need to modify anything else.

Reply:
The other ones should stay same and aren't in axiom-tools.
`numLookupBits` should always be `k - 1`, and when the circuit does auto_config, it should select a new `k` optimally.